### PR TITLE
Update tooltips for the Equation Button and Style Picker

### LINKED ISSUE #1160: Update tooltips for the EquationButton and add them to the Color picker colors
Update the equation button automation name and tooltip to use different resource items and add tooltips to the colors in the style picker.

### PR DESCRIPTION
**Pull request by sanderl**
### Description of the changes:
- Updates the equation button automation name and tooltip to use different resource items
- Add tooltips to the colors in the style picker
- Fixed the issue where the white color has the wrong automation name

### How changes were validated:

- manually with narrator and validated the tooltips

### CODE REVIEW

File: src/Calculator/Resources/en-US/Resources.resw [hunk 1]
```diff
@@ -4614,4 +4623,40 @@
     <value>Function Analysis Equation Box</value>
     <comment>This is the automation name text for the equation box in the function analsis panel</comment>
```
Comment (ItzLevvie) [inline src/Calculator/Resources/en-US/Resources.resw:4624]:
```suggestion
    <comment>This is the automation name text for the equation box in the function analysis panel</comment>
```

File: src/Calculator/EquationStylePanelControl.xaml [hunk 2]
```diff
@@ -80,6 +80,7 @@
                              Fill="{x:Bind}"
                              StrokeThickness="0"
                              AutomationProperties.Name="{x:Bind local:EquationStylePanelControl.GetColorAutomationName((Brush))}"
+                             ToolTipService.ToolTip="{x:Bind local:EquationStylePanelControl.GetColorAutomationName((Brush))}"
```
Comment (joseartrivera) [inline src/Calculator/EquationStylePanelControl.xaml:83]: All of the colors show up as black
image

File: src/Calculator/EquationStylePanelControl.xaml.cpp [hunk 3]
```diff
@@ -145,61 +145,81 @@ String ^ EquationStylePanelControl::GetColorAutomationName(Brush ^ brush)
     auto resourceLoader = AppResourceProvider::GetInstance();
     auto color = static_cast<SolidColorBrush ^>(brush);
 
-    if (color == safe_cast<SolidColorBrush ^>(Application::Current->Resources->Lookup(L"EquationBrush1")))
+    auto lightDictionary = static_cast<ResourceDictionary ^>(Application::Current->Resources->ThemeDictionaries->Lookup(L"Light"));
```
Comment (joseartrivera) [inline src/Calculator/EquationStylePanelControl.xaml.cpp:148]: Is it intended for high contrast to have all of the labels be Black also? Maybe Color 1, Color 2, etc might be better?

**Pull request merged**